### PR TITLE
Let swupdate Makefile know about MTD versions.:

### PR DESCRIPTION
**Pull request by Ealdwulf**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR makes the build of swupdate insensitive to the version of mtd-utils, since later versions of openwrt use a newer mtd-utils

## Manual test
- [x] 👍 yes
- [ ] 🙅 no

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by cpunt — APPROVED** [review]
LGTM! Once it’s squashed and merged into main, we'll need to create a new tag/release at the tip of main so we can build from it

**Comment by Ealdwulf** [comment]
> LGTM! Once it’s squashed and merged into main, we'll need to create a new tag/release at the tip of main so we can build from it

Should I merge this no, or do you need to schedule time to dealwith this @cpunt ?

**Pull request merged**
